src/i18n/langResources/lang.de.ts

```diff
@@ -1,38 +1,38 @@
 import { ILangResource } from '../../models/i18n';
 
 export const langDe: ILangResource = {
-  newVersionMessage: 'Willkommen zur neuen Version von vscode-icons!',
-  seeReleaseNotes: 'Was wurde geändert? (Englisch)',
-  dontShowThis: 'Diese Nachricht beim nächsten Update nicht mehr anzeigen',
-  seeReadme: 'Etwas über die Erweiterung lernen (Englisch)',
-  welcomeMessageBegin: 'vscode-icons benutzt nun die offizielle API. Aktiviere die Icons unter ',
-  welcomeMessageEnd: 'und wähle VSCode Icons aus.',
+  newVersionMessage: 'Willkommen zur neuen Version von vscode-icons.',
+  seeReleaseNotes: 'Informationen über die letzten Änderungen (en englisch)',
+  dontShowThis: 'Zeige mir diese Nachricht nicht das nächste Mal',
+  seeReadme: 'Erfahren Sie mehr über diese Erweiterung (en englisch)',
+  welcomeMessageBegin: 'vscode-icons benutzt nun die offizielle API. Gehen Sie zu ',
+  welcomeMessageEnd: 'und wählen Sie VSCode Icons, um zu aktivieren.',
   activationPath: {
     darwin: 'Code > Einstellungen > Datei-Symboldesign ',
     linux: 'Datei > Einstellungen > Datei-Symboldesign ',
     win32: 'Datei > Einstellungen > Datei-Symboldesign ',
   },
-  aboutOfficialApi: 'Mehr über Datei und Ordner Icons herausfinden (Englisch)',
-  learnMore: 'Mehr herausfinden!',
+  aboutOfficialApi: 'Erfahren Sie mehr über Datei und Ordner Icons (en englisch)',
```

Comment:
Yeah. Correct is `am englisch`. Knowing too many spoken languages is like knowing too many programming languages. At some point, you end up mixing syntax.